packages/sst/support/remix-site-function/regional-server.js

```diff
@@ -153,10 +153,22 @@ const createApigHandler = (build) => {
   const requestHandler = createNodeRequestHandler(build, process.env.NODE_ENV);
 
   return async (event) => {
+    if (event.type === "warmer") {
+      return formatWarmerResponse(event);
+    }
+
     const request = convertApigRequestToNode(event);
     const response = await requestHandler(request);
     return convertNodeResponseToApig(response);
   };
 };
 
+function formatWarmerResponse(event) {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve({ serverId });
```

Comment:
`serverId` here is undefined. Looking through the code, I don't see where it's supposed to come from. Is it `event.warmerId`?